doc/source/scalability/innodb_io.rst

```diff
@@ -46,26 +26,24 @@ If :variable:`innodb_use_global_flush_log_at_trx_commit` is set to ``1`` (True),
    :Dyn: No
    :vartype: Enumeration
    :default: ``fdatasync``
-   :allowed: ``fdatasync``, ``O_DSYNC``, ``O_DIRECT``, ``O_DIRECT_NO_FSYNC``, ``ALL_O_DIRECT``
+   :allowed: ``fdatasync``, ``O_DSYNC``, ``O_DIRECT``, ``O_DIRECT_NO_FSYNC``
```

Comment:
I'd remove this completely. Parallel doublewrite buffer flushing is discussed there